lxd/shared.go

```diff
@@ -64,6 +64,26 @@ func resourceLxdConfigMap(c interface{}) map[string]string {
 	return config
 }
 
+func resourceLxdConfigMapAppend(c interface{}, config map[string]string, namespace string) map[string]string {
```

Comment:
Possible nit-picking, but would this be better if it was `resourceLxdConfigMapAppend(config, c, namespace)`?

`config` is the map being appended to, `c` is the smaller set with new data.